cpp/arcticdb/util/error_code.hpp

```diff
@@ -97,10 +98,16 @@ constexpr ErrorCategory get_error_category(ErrorCode code) {
     return static_cast<ErrorCategory>(static_cast<BaseType>(code) / error_category_scale);
 }
 
+struct ArcticMarkerException : public std::runtime_error {
```

Comment:
What does marker mean in this context?

Reply:
Hmm this is a bit of a hangover from an earlier iteration that is a bit obsolete now. Let me try and clean up a bit, thanks